### ISSUE TITLE
Make reboot of devices after each run optional

### ISSUE THREAD

**Issue by stefanhoth** [ssmiech/rubotium#24]
Before I create a PR with that I was wondering if there's anything that fundamentally speaks against making the reboot of a device after test optional ( https://github.com/ssmiech/rubotium/blob/master/lib/rubotium.rb#L92).

I assume this command is used to ensure a "clean" state of the device but it also blocks it for execution for the period of the reboot. I'd like to disable this until we realize that we indeed need to force that reboot.

**Comment by ssmiech** [comment]
Good point. It should be optional. We enforced it in SC and I think I accidentally pushed it to  the repo.
